app/brave_settings_strings.grdp

```diff
@@ -1090,13 +1090,13 @@
     Improve search suggestions
   </message>
   <message name="IDS_SETTINGS_BRAVE_SEARCH_ENGINES_SEARCH_SUGGEST_DESC" desc="The desc of the option to enable/disable sending omnibox input to the user's default search engine to get additional suggestions.">
-    When on, Brave sends what you type in the address bar or search box to your default search engine to get better suggestions. When off, Brave only shows suggestions generated on your device. This is always off in private windows.
+    When on, Brave sends what you type in the address bar or search box to your default search engine to get better suggestions. When off, Brave only shows suggestions generated on your device. This is always off in private windows
```

Comment:
The period here can stay. "This is always off in private windows."

Reply:
Restored in https://github.com/brave/brave-core/pull/30665/commits/fde7d0a637f0d71f6bf63ce423890db27ae10b12